### ISSUE TITLE
Launcher load in compress window form.

### ISSUE THREAD

**Issue by Zolrex** [AsteriskAmpersand/Ice-Stable#9]
![Screenshot 2023-10-19 231424](https://github.com/AsteriskAmpersand/Ice-Stable/assets/110205371/1f9703c1-1a81-4333-87e3-cc0cd9f52a37)

it's load like this...seems some issue with resolution of app.

**Comment by Keitren** [comment]
I have this same issue playing with a resoluton of 3840x2160.

**Comment by AsteriskAmpersand** [comment]
What does Compress Window form even mean?

ICE Manager has minimum window size and parameters. That looks like you are using one of the accessibility option for text size to its very extreme.

**Comment by Zolrex** [comment]
> I have this same issue playing with a resoluton of 3840x2160.

yes,seems every resolution option above 1080p,making launcher window looks ummm kinda "compressed"(like it's showed on screenshot above),it's like all its functions buttons became like very tiny and messy along with text there..

**Comment by AsteriskAmpersand** [comment]
Seems like a problem with the way Qt deals with other resolutions. Will investigate more.
